src/Microsoft.OData.Client/ALinq/ResourceBinder.cs

```diff
@@ -2910,6 +2988,46 @@ internal static void ValidateExpandPath(Expression input, DataServiceContext con
 
                 throw new NotSupportedException(Strings.ALinq_InvalidExpressionInNavigationPath(input));
             }
+
+            /// <summary>
+            /// Checks whether the specified <paramref name="expr"/> is a valid aggregate expression.
+            /// </summary>
+            /// <param name="expr">The aggregate expression</param>
+            internal static void ValidateAggregateExpression(Expression expr)
+            {
+                MemberExpression memberExpr = StripTo<MemberExpression>(expr);
+                

```

Comment:
What does `StripTo` do?

Reply:
@KenitoInc It strips the expression of intermediate expressions like Expression.Convert() and Expression.Quote(). See more [here](https://github.com/OData/odata.net/blob/d853504765436a1cd7b8df2b56598433e53d101b/src/Microsoft.OData.Client/ALinq/ResourceBinder.cs#L1180)